src/radical/saga/adaptors/slurm/slurm_job.py

```diff
@@ -296,6 +375,9 @@ def init_instance(self, adaptor_state, rm_url, session):
         self.rm      = rm_url
         self.session = session
 
+        self.mt = _job_state_monitor(job_service=self)
+        self.mt.start()
```

Comment:
it never stops here

Reply:
That's ok - the thread is configured as daemon thread and will die with the process.